zeyple/zeyple.py

```diff
@@ -24,24 +26,39 @@
 class Zeyple:
     """Zeyple Encrypts Your Precious Log Emails"""
 
-    def __init__(self):
-        self._load_configuration()
+    def __init__(self, config_fname='zeyple.conf'):
+        self.config = self.load_configuration(config_fname)
 
-        log_file = self._config.get('zeyple', 'log_file')
+        log_file = self.config.get('zeyple', 'log_file')
         logging.basicConfig(
             filename=log_file, level=logging.DEBUG,
             format='%(asctime)s %(process)s %(levelname)s %(message)s'
         )
         logging.info("Zeyple ready to encrypt outgoing emails")
 
-        # tells gpgme.Context() where are the keys
-        os.environ['GNUPGHOME'] = self._config.get('gpg', 'home')
+    @property
+    def gpg(self):
+        protocol = gpgme.PROTOCOL_OpenPGP
 
-    def process_message(self, message, recipients):
+        if self.config.has_option('gpg', 'executable'):
+            executable = self.config.get('gpg', 'executable')
+        else:
+            executable = None  # Default value
+
+        home_dir = self.config.get('gpg', 'home')
+
+        ctx = gpgme.Context()
+        ctx.set_engine_info(protocol, executable, home_dir)
```

Comment:
Nice! I remember having trouble with that and had to resort injecting the `GNUPGHOME` env variable which is ugly.